robocup_ws/src/Planner/src/player_controller.py

```diff
@@ -0,0 +1,128 @@
+from game_interfaces.msg import PlayerCommand
+import numpy as np
+import matplotlib.pyplot as plt
+from BasicCommonActions.go_to_action import simple_go_to_action
+from game_interfaces.msg import Position
+from Planner.src.brachistochrone import cycloid
+from BasicCommonActions.go_to_action import go_to_fast
+import math
+
+
+class PlayerController:  #(Robot)
+    def __init__(self, player_id):
+        self.read = True
+        self.player = player_id
+        self.points_to_visit = []
+        self.current_goal = 0
+        self.goal_threshold = 0.1
+
+    def has_ball(self, game_info):
```

Comment:
Generalise and move to
https://github.com/szgula/UofG_Robotics_TDP/blob/a7fddf38f5438ca2089b1e7bdb62ef6a77113c7a/robocup_ws/src/Planner/src/BasicCommonActions/plan_supporting_functions.py#L6

Probably the good should return: 
```
-1: opponent team has ball
0: no team has a ball
1: out team has a ball
```